docs/resources/data_source.md

```diff
@@ -131,8 +150,9 @@ Optional:
 - **password** (String, Sensitive) (All) Password to use for authentication.
 - **private_key** (String, Sensitive) (Stackdriver) The service account key `private_key` to use to access the data source.
 - **secret_key** (String, Sensitive) (CloudWatch) The secret key to use to access the data source.
+- **sigv4_access_key** (String, Sensitive) (Elasticsearch and Prometheus) SigV4 access key. Required when using 'keys' auth provider.
+- **sigv4_secret_key** (String, Sensitive) (Elasticsearch and Prometheus) SigV4 secret key. Required when using 'keys' auth provider.
 - **tls_ca_cert** (String, Sensitive) (All) CA cert for out going requests.
 - **tls_client_cert** (String, Sensitive) (All) TLS Client cert for outgoing requests.
 - **tls_client_key** (String, Sensitive) (All) TLS Client key for outgoing requests.
 
```

Comment:
`tfplugindocs` wants this space here. Adding back so `go generate` doesn't produce a diff.

```suggestion


```